bluesky/callbacks/zmq.py

```diff
@@ -0,0 +1,257 @@
+import ast
+import asyncio
+import multiprocessing
+import os
+import socket
+import time
+from ..run_engine import Dispatcher, DocumentNames
+from ..utils import expiring_function
+
+
+class Publisher:
+    """
+    A callback that publishes documents to a 0MQ proxy.
+
+    Parameters
+    ----------
+    RE : ``bluesky.RunEngine``
```

Comment:
Is there a way we can drop this?

I don't think so because it needs to know how to un-subscribe it's self